catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/util/actions/CustomActions.java

```diff
@@ -55,4 +55,24 @@ public void perform(UiController uiController, View view) {
 			}
 		};
 	}
+
+	public static ViewAction clickChildViewWithId(final int id) {
```

Comment:
this is not necessary, once above "wrapper" funcitonality has been moved over to UserDataItemRVInteractionWrapper and there you can just use onChildView(withId(....